AccountControl/Infrastructure/Middleware/GlobalExeptionMiddleware.cs

```diff
@@ -0,0 +1,44 @@
+﻿using AccountControl.Infrastructure.Interfaces;
+using System.Net;
+using System.Text.Json;
+
+namespace AccountControl.Infrastructure.Middleware
+{
+    public class GlobalExeptionMiddleware
```

Comment:
Custom middleware mustn't be on Infrastructure level, should be on the same level as controllers (called Presentation in your case)